src/base/LockToGovernBase.sol

```diff
@@ -36,19 +37,20 @@ abstract contract LockToGovernBase is ILockToGovernBase, IMembership, ERC165Upgr
     }
 
     /// @inheritdoc ILockToGovernBase
-    function token() external view returns (IERC20) {
-        return lockManager.token();
-    }
-
-    /// @inheritdoc ILockToGovernBase
-    function underlyingToken() external view returns (IERC20) {
-        return lockManager.underlyingToken();
+    function token() public view returns (IERC20) {
+        return IERC20(lockManager.token());
     }
 
     /// @inheritdoc IMembership
     function isMember(address _account) external view returns (bool) {
         if (lockManager.lockedBalances(_account) > 0) return true;
-        else if (lockManager.token().balanceOf(_account) > 0) return true;
+
+        IERC20 _token = token();
+        if (address(_token) != address(0)) {
+            if (_token.balanceOf(_account) > 0) return true;
+        }
+        /// @dev When there is no token, we are not considering the native token as a way of membership
```

Comment:
Does this mean we should support native? There's a strong argument IMO for not doing so.